src/run/runner/valgrind/measure.rs

```diff
@@ -51,6 +51,7 @@ pub fn measure(
     cmd.arg(ARCH).arg("-R");
     // Configure the environment
     cmd.envs(BASE_INJECTED_ENV.iter())
+        .env("CODSPEED_PROFILE_FOLDER", profile_folder.to_str().unwrap())
```

Comment:
refactor with `CODSPEED_RUNNER_MODE` so that we don't duplicate this logic with the other mode